helix-core/src/main/java/org/apache/helix/controller/stages/AttributeName.java

```diff
@@ -45,6 +45,9 @@
   /** This is the cluster manager's session id when event is received. */
   EVENT_SESSION,
 
+  /** Represents cluster's management mode status. Used only in management mode pipeline */
+  MANAGEMENT_MODE,
```

Comment:
I would suggest this to be CLUSTER_STATUS instead of MANAGEMENT_MODE. status may be a more generic type can share with multiple pipelines.